Gemfile

```diff
@@ -33,9 +33,9 @@ gem 'gettext_i18n_rails', '~> 1.8'
 gem 'rails-i18n', '~> 6.0'
 gem 'i18n', '~> 1.1'
 gem 'logging', '>= 1.8.0', '< 3.0.0'
-gem 'fog-core', '2.1.0'
+gem 'fog-core', '>= 2.1.0', '< 3.0.0'
 gem 'net-scp'
-gem 'net-ssh', '4.2.0'
+gem 'net-ssh', '>= 4.2.0', '< 7.0.0'
```

Comment:
Like this @ekohl? 6.1 is out, so we are going to test it with that, so we should just prevent installing new major version, right?

Reply:
Exactly.